Subway/Subway/Program.cs

```diff
@@ -1,16 +1,18 @@
 ﻿using System;
-using System.Collections.Generic;
 
 namespace Subway
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Subway subway = new Subway("D:\\Subway\\Subway\\Subway\\StationList.txt");
-            string path = subway.GetPath("AjaxRapids", "GoF Gardens");
+            Subway subway = new Subway(Loader.Load("D:\\Subway\\Subway\\Subway\\StationList.txt"));
```

Comment:
How extensible is static class? What should I do if I want to add another data source?